internal/stores/metadata.go

```diff
@@ -911,7 +916,7 @@ func removeObject(tx *gorm.DB, key string) error {
 // removeContract removes a contract from the store.
 func removeContract(tx *gorm.DB, id fileContractID) error {
 	return tx.
-		Where(&dbContract{FCID: id}).
+		Where(&dbContract{ContractCommon: ContractCommon{FCID: id}}).
```

Comment:
🤷🏻‍♂️ kind of sad but perhaps it's not too bad

Reply:
renaming it to `ContractMetadata` would "fix" it... let me know if that is what you prefer